torch/csrc/jit/passes/quantization.cpp

```diff
@@ -118,6 +118,7 @@ std::vector<std::string> _single_input_general_aten_funcs = {
     "upsample_bicubic2d",
     "dropout",
     "reshape",
+    "resize_",
```

Comment:
what about "resize"?

Reply:
`resize` is a python module, and script does not see it:

```python
class Model(torch.nn.Module):
  def __init__(self, sz):
    super(Model, self).__init__()
    self.sz = sz
  def forward(self, x):
    return x.resize(self.sz)
torch.jit.script(Model(42))
```

will throw an error:

```console
RuntimeError: 
Tried to access nonexistent attribute or method 'resize' of type 'Tensor'.:
  File "<ipython-input-11-85bfdcc7b088>", line 6
  def forward(self, x):
    return x.resize(self.sz)
           ~~~~~~~~ <--- HERE
```

Reply:
I see, looks like non-inplace resize is deprecated: https://github.com/pytorch/pytorch/blob/master/torch/tensor.py#L351

could you add a comment here for it?